src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/5.diff.sql

```diff
@@ -0,0 +1,668 @@
+/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+******************************************************************************************/
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+
+IF NOT EXISTS 
+(
+    SELECT *
+    FROM    sys.columns
+    WHERE   NAME = 'TagValueUTC'
+            AND object_id = OBJECT_ID('dbo.ExtendedQueryTagDateTime')
+)
+BEGIN
+    ALTER TABLE dbo.ExtendedQueryTagDateTime
+    ADD TagValueUTC DATETIME2(7) NULL
+END
+
+CREATE UNIQUE CLUSTERED INDEX IXC_ExtendedQueryTagDateTime ON dbo.ExtendedQueryTagDateTime
+(
+    TagKey,
+    TagValue,
+    TagValueUTC,
```

Comment:
Can you give me 2 example of how will the QIDO generated SQL looks like for using this new column? Based on that con you confirm if this is the right index?

Reply:
So for now, we don't need TagValueUTC. I placed it considering that we will eventually support offset in QIDO. Once that is implemented, TagValueUTC will help in the indexing. I can remove it for now and add it once we implement offset support in QIDO.

Reply:
Sounds good. Just a FYI, the way you had placed the TagValueUTC will probably not work. Please read how composite indexes work in SQL